install.sh

```diff
@@ -104,7 +104,14 @@ install_deps_osx() {
 
     echo -e "\nInstalling dependencies ...\n"
     brew update
-    brew install openssl libsodium
+
+    if [[ $(brew info openssl --json | jq ".[].installed|length > 0") = false ]]; then
```

Comment:
The installer should not rely on `jq` being installed on the user systems, most of the times it's not

Reply:
c6acf82